Kconfig.zephyr

```
@@ -747,6 +747,7 @@ config BUILD_OUTPUT_STRIP_PATHS
 config CHECK_INIT_PRIORITIES
 	bool "Build time initialization priorities check"
 	default y
+	depends on !NATIVE_LIBRARY
```

nothing wrong on this line, but could you add a safe-guard here, so that this not available for armclang.

Something like:
```suggestion
	depends on !NATIVE_LIBRARY
	depends on "$(ZEPHYR_TOOLCHAIN_VARIANT)" != "armclang"
```

---

Aha, thanks, sorry somehow I could not find a Kconfig option for this, that works.